hw/top_earlgrey/ip/flash_ctrl/data/autogen/flash_ctrl.hjson

```diff
@@ -423,8 +425,8 @@
             hwaccess: "hrw",
             name: "START",
             desc: '''
-              Start flash transaction.  This bit shall only be set after the other fields of the
-              CONTROL register and ADDR have been programmed
+              Start flash transaction.  This bit shall only be set at the same time or after the other
+              fields of the !!CONTROL register and !!ADDR have been programmed.
```

Comment:
```suggestion
              Start flash transaction.  This bit shall not be set before the other
              fields of the !!CONTROL register and !!ADDR have been programmed.
```

Reply:
actually this one i probably won't change. The DV people looking at this specifically want the notion of "at the same time or after"

Reply:
NP